### PR TITLE
update prokka ad roary

### PR DESCRIPTION
**Pull request by bgruening**
@Slugger70 can you look at this one, please. The only big change that I wanted to do is adding `min="2"` to roary's gff inputs. We got a few error reports from users that only supply one GFF file.

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Review by Slugger70 — APPROVED** [review]

File: tools/prokka/prokka.xml [hunk 1]
```diff
@@ -74,8 +78,8 @@ $input
     ]]></command>
     <inputs>
         <param name="input" type="data" format="fasta" label="Contigs to annotate" help="FASTA format" />
-        <param name="locustag" type="text" value="" label="Locus tag prefix (--locustag)" />
-        <param name="increment" type="integer" value="1" min="1" optional="true" label="Locus tag counter increment (--increment)" />
+        <param argument="locustag" type="text" value="" label="Locus tag prefix" />
```
Comment (bernt-matthias) [inline tools/prokka/prokka.xml:81]: Can you add the dashes in the argument attribute?

File: tools/roary/roary.xml [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+        <requirement type="package" version="@TOOL_VERSION@">roary</requirement>
     </requirements>
 
     <command detect_errors="exit_code"><![CDATA[
         #set $filenames = list()
         #for $gff in $gff_input.gffs
-          cp '$gff' '${gff.element_identifier}.gff' &&
-          #set $filename = str($gff.element_identifier) + '.gff'
-          $filenames.append(str($filename))
+            cp '$gff' '${gff.element_identifier}.gff' &&
```
Comment (bernt-matthias) [inline tools/roary/roary.xml:14]: There are some additional spaces, or?

File: tools/prokka/prokka.xml (hunk 1 above)
Comment (bgruening) [reply tools/prokka/prokka.xml]: done :)

File: tools/roary/roary.xml (hunk 2 above)
Comment (bgruening) [reply tools/roary/roary.xml:14]: VSCode did the reformating, I think this is ok. Or what do you mean?

**Comment by bgruening** [comment]
@bernt-matthias ping :)

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**